source/js/fellowships.js

```diff
@@ -171,7 +172,7 @@ function injectReactComponents(env) {
 
   // Fellows on Fellows Directory page
   if (document.getElementById(`fellows-directory-featured-fellows`)) {
-    renderFellowsOnDirectoryPage();
+    renderFellowsOnDirectoryPage(env.FELLOWSHIPS_DIRECTORY_TYPES.slice(0));
```

Comment:
i wonder if we really need to create a new copy of `env.FELLOWSHIPS_DIRECTORY_TYPES` here?